02.calendar/cal.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+options = ARGV.getopts('y:','m:')
```

Comment:
慣習として、`require`をいくつか書いた後には空行を入れておいた方が良いと思います。
```suggestion
require 'date'

options = ARGV.getopts('y:','m:')
```